hazelcast/src/main/java/com/hazelcast/internal/nio/ascii/RestApiFilter.java

```diff
@@ -46,7 +46,17 @@ public void filterConnection(String commandLine, ServerConnection connection) {
         RestEndpointGroup restEndpointGroup = getEndpointGroup(commandLine);
         if (restEndpointGroup != null) {
             if (!restApiConfig.isGroupEnabled(restEndpointGroup)) {
-                connection.close("REST endpoint group is not enabled - " + restEndpointGroup, null);
+                String name = restEndpointGroup.name();
+                connection.close("REST endpoint group is not enabled - " + restEndpointGroup
+                        + ". To enable it, please do one of the following:\n"
```

Comment:
would be good to add also the env. property version. that's useful when running Hazelcast in a container. 
`HZ_NETWORK_REST-API_ENDPOINT-GROUPS_CP_ENABLED=true`